config/simorgh/simorgh.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+  baseUrl: 'http://localhost:7080',
+  paths: [
+    '/news/articles/c0000000025o',
+    '/news/articles/c0000000027o'
+  ],
+  LIGHTHOUSE_OPTS: {
```

Comment:
Uppercased names like this are normally reserved for constants, rather than properties in an object. Can we rename it to `lighthouseOptions` or such like?